spec/system/composer/category_templates_spec.rb

```diff
@@ -195,4 +213,24 @@
       "Bruce Wayne",
     )
   end
+
+  it "creates a post with an upload field" do
+    topic_title = "Bruce Wayne's Medication"
+
+    category_page.visit(category_with_upload_template)
+    category_page.new_topic_button.click
+    attach_file "upload-your-prescription-uploader",
```

Comment:
I think it'll be helpful to test multiple file uploads as well in our test case.

Reply:
We also have various conditions like audio, video and images with width to test for.